src/main/resources/schemas/non-strategic/interim_applications_chancery_division_daily_cause_list.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Root",
+  "type": "object",
+  "required": [
+    "hearingList",
+    "openJusticeStatementDetails"
+  ],
+  "properties": {
+    "hearingList": {
+      "title": "Hearing list",
+      "type": "array",
+      "items": {
+        "title": "Hearing",
+        "type": "object",
+        "required": [
+          "judge",
+          "time",
+          "venue",
+          "hearingType",
```

Comment:
The schema in the ticket says the column is Type and not heatingType. Think we should follow that as this have been agreed between Kim and the court admins